{"name": "Проверка получения ошибки аутентификации курьера при вводе невалидных данных", "status": "passed", "description": "В тест по очереди передаются наборы данных с несуществующим логином или неверным паролем. Проверяются код и тело ответа.", "parameters": [{"name": "nonexistent_credentials", "value": "{'login': 'Ilya1990', 'password': '123456'}"}], "start": 1723036926294, "stop": 1723036926594, "uuid": "422927cc-39a6-41fb-a176-f0974062a0bb", "historyId": "7ce53eab6ce1d2c17e2e0268189a0b6c", "testCaseId": "199be2f2af7455b14ed4795a4ee970da", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-82VEM3U"}, {"name": "thread", "value": "12784-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}